lms/templates/course_modes/_upgrade_button.html

```diff
@@ -10,10 +10,10 @@
 <li class="action action-select">
     <input type="hidden" name="contribution" value="${price_before_discount or min_price}" />
     % if content_gating_enabled or course_duration_limit_enabled:
-        <button type="submit" name="verified_mode">
+        <button id="track_sel_upgrade" type="submit" name="verified_mode">
```

Comment:
personally i prefer not having abbreviations, so i think track_selection_upgrade would be better, but you don't have to address this comment

Reply:
> personally i prefer not having abbreviations, so i think track_selection_upgrade would be better, but you don't have to address this comment

No problem- you couldn't have picked an easier one to retest :) Update made!